### ISSUE TITLE
HOW CAN I INSERT ANSWER OF PARTICULAR QUESTION? 

### ISSUE THREAD

**Issue by disheet** [MycroftAI/mycroft-core#593]
I have few question with answer and i wanted to insert that one only.So how can i do that? Actually, I want my Mycroft to give answer of particular question which one i wanted.....

**Comment by kfezer** [comment]
I'm not sure what exactly you mean, could you rephrase your question?


Generally, we like using issues to show bugs and enhancement to the code, not answering questions. The best place for that is going to be the forums:
https://community.mycroft.ai/
or slack:
https://mycroft-ai-slack-invite.herokuapp.com/

I'm going to close this issue for now.

**Comment by disheet** [comment]
I mean i wanted add my own skills ...so Is it possible?

**Comment by kfezer** [comment]
Yes you can. You add them to the skills directory and they get added automatically. For that, I'd check out our documentation.
https://docs.mycroft.ai/
If you have any questions after taking a look, feel free to post on the community or contact us on slack!

**Comment by disheet** [comment]
thank you sir.

And more thing i changed wake word as per requirement. But after that
without speaking wake word it was continuously working...means It is not
waiting for wake word..."hy mycroft"....why its happening?Because of
threshold or some other thing???/

**Comment by kfezer** [comment]
I'm not sure. I haven't seen this issue before. I would recommend asking it in slack or the community and see if anyone else has had a similar problem.

**Comment by disheet** [comment]
ok sir thank you for your help

**Comment by disheet** [comment]
i wanted to changes answer of " hy mycroft" , where can i find that
directory/file.....from last 2-3 days i am searching but i didn't got
success...

**Comment by kfezer** [comment]
It's in the configuration file. There's a FAQ int he documentation:

https://docs.mycroft.ai/installing.and.running/FAQ
